--- src/CMakeLists-orig.txt	2021-03-09 17:50:03.068035926 +0100
+++ src/CMakeLists.txt	2021-03-09 17:52:58.208036913 +0100
@@ -41,6 +41,11 @@
 		message(STATUS "cURL support enabled.")
 		set(USE_CURL TRUE)
 	endif()
+	find_package(OpenSSL)
+	if (NOT OPENSSL_FOUND)
+		message(WARNING "OpenSSL not found.")
+	endif()
+
 else()
 	mark_as_advanced(CLEAR CURL_LIBRARY CURL_INCLUDE_DIR)
 endif()
@@ -661,6 +666,13 @@
 			${PROJECT_NAME}server
 			${CURL_LIBRARY}
 		)
+		if(OPENSSL_FOUND)
+			target_link_libraries(
+				${PROJECT_NAME}server
+				${OPENSSL_CRYPTO_LIBRARY}
+				${OPENSSL_SSL_LIBRARY}
+			)
+		endif()
 	endif()
 endif(BUILD_SERVER)
 
