--- lib/jsoncpp/jsoncpp-orig.cpp	2021-03-09 17:23:08.768026834 +0100
+++ lib/jsoncpp/jsoncpp.cpp	2021-03-09 17:23:25.960026931 +0100
@@ -3224,7 +3224,7 @@
     return value_.uint_ != 0;
   case realValue: {
     // According to JavaScript language zero or NaN is regarded as false
-    const auto value_classification = std::fpclassify(value_.real_);
+    const auto value_classification = fpclassify(value_.real_);
     return value_classification != FP_ZERO && value_classification != FP_NAN;
   }
   default:
